generators/app/templates/src/app/components/Button/index.js

```diff
@@ -42,12 +27,7 @@ function Button({
       </Link>
     </div>
   ) : (
-    <button
-      type={type}
-      style={[styles.base, style, props.style]}
-      disabled={disabled}
-      onClick={onPress}
-    >
+    <button type={type} style={[styles.base, style, props.style]} disabled={disabled} onClick={onPress}>
```

Comment:
Shouldn't this receive a spread of `props` ?

Reply:
This'll be erased in a future PR